functionaltest-jenkins-plugin/build.gradle

```diff
@@ -70,4 +70,6 @@ dependencies {
     implementation 'io.gsonfire:gson-fire:1.8.5'
     implementation 'org.openapitools:jackson-databind-nullable:0.2.2'
     implementation "jakarta.annotation:jakarta.annotation-api:2.0.0"
+    implementation 'javax.annotation:javax.annotation-api:1.3.2'
+    implementation 'javax.xml.bind:jaxb-api:2.3.1'
```

Comment:
Why do we need that?

Reply:
https://app.circleci.com/pipelines/github/stackrox/jenkins-plugin/1107/workflows/46558806-ba47-45f7-9146-1afba31ec939/jobs/3002
&
https://app.circleci.com/pipelines/github/stackrox/jenkins-plugin/1106/workflows/63ee3fa3-5ef4-4a4b-b831-5bb353790335/jobs/2999

Reply:
OK, so I guess it's related to JDK version used in container